inc/taskjob.class.php

```diff
@@ -530,19 +447,18 @@ function dropdownAction($myname, $actiontype, $method, $actiontypeid, $value=0,
     * @return array [id integed agent id] => $name value agent name
     */
    function getAgents($module) {
-
-      $array = [];
-      $array[".1"] = " ".__('Auto managenement dynamic of agents', 'fusioninventory');
```

Comment:
Another old thing not longer used?

Reply:
yes, it was an old task feature that has been deprecated during the task rework